### PR TITLE
Add new person reid-model 0300

### PR DESCRIPTION
**Pull request by DmitriySidnev**

### CODE REVIEW

File: tools/accuracy_checker/configs/person-reidentification-retail-0300.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
+models:
+  - name: person-reidentification-retail-0300
```
Comment (eaidova) [inline tools/accuracy_checker/configs/person-reidentification-retail-0300.yml]: @DmitrySidnev,  I right understand, is it replacemdent for person-reidentification-retail-0200?

Comment (eaidova) [reply tools/accuracy_checker/configs/person-reidentification-retail-0300.yml]: if yes, please also remove config for 0200

File: tools/accuracy_checker/configs/person-reidentification-retail-0300.yml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+      - framework: dlsdk
+        tags:
+          - FP32-INT8
+        model:   intel/person-reidentification-retail-0300/FP32-INT8/person-reidentification-retail-0300.xml
+        weights: intel/person-reidentification-retail-0300/FP32-INT8/person-reidentification-retail-0300.bin
+        adapter: reid
+
+    datasets:
+      - name: market1501
+
```
Comment (eaidova) [inline tools/accuracy_checker/configs/person-reidentification-retail-0300.yml]: please pay attention that you read images with opencv (BGR) and then resize them with pillow. Is this preprocessing correct?

File: tools/accuracy_checker/configs/person-reidentification-retail-0300.yml (hunk 1 above)
Comment (vladimir-dudnik) [reply tools/accuracy_checker/configs/person-reidentification-retail-0300.yml]: removed

Comment (vladimir-dudnik) [reply tools/accuracy_checker/configs/person-reidentification-retail-0300.yml]: removed

**Comment by vladimir-dudnik** [comment]
Jenkins please retry a build

**Comment by DmitriySidnev** [comment]
@vladimir-dudnik, @eaidova, I saw that your comments already fixed. Should I do something else?

**Comment by vladimir-dudnik** [comment]
@DmitriySidnev no, thanks, I think it is ready to merge

**Comment by vladimir-dudnik** [comment]
Jenkins please retry a build

**Pull request merged**